linode_api4/linode_client.py

```diff
@@ -11,16 +12,19 @@
 from linode_api4.objects import *
 from linode_api4.objects.filtering import Filter
 
-from .common import load_and_validate_keys, SSH_KEY_TYPES
+from .common import SSH_KEY_TYPES, load_and_validate_keys
 from .paginated_list import PaginatedList
 
+if TYPE_CHECKING:
+    from linode_api4 import LinodeClient
```

Comment:
I might be misunderstanding this, but could we use [postponed annotations](https://stackoverflow.com/a/61544901)?

Reply:
Yea, you are right. I didn't notice `LinodeClient` is also defined in this file.

Reply:
Thanks, and it is now fixed.